main/src/ca/uwaterloo/flix/language/phase/monomorph/Specialization.scala

```diff
@@ -122,7 +122,7 @@ object Specialization {
     *   - No type aliases
     *   - Equivalent types are uniquely represented (e.g. fields in records types are alphabetized)
     */
-  private case class StrictSubstitution(s: Substitution) {
+  protected[monomorph] case class StrictSubstitution(s: Substitution) {
```

Comment:
Maybe StrictSubstitution should go in StrictSubstitution.scala in the package? At least it could...